cmd/synchronization/app/options.go

```diff
@@ -43,7 +45,17 @@ func (p *Providers) UnmarshalJSON(data []byte) error {
 	}
 	depURL, err := url.Parse(rawProviders.Deployinator)
 	p.DeployinatorURL = depURL
-	return errors.Wrap(err, "unable to parse Deployinator URL")
+	if err != nil {
+		return errors.Wrap(err, "unable to parse Deployinator URL")
+	}
```

Comment:
I am not sure why this always returned an error and I cannot identify where this function is called.

Reply:
seems like a bug, not sure how did it work

Reply:
it should be implicitly called in https://github.com/atlassian/voyager/blob/master/cmd/synchronization/app/options.go#L67